### PR TITLE
Background wave

### PR DESCRIPTION
**Pull request by lianthompson**
https://5b7f3eb302ed8325f5fc73a1--relaxed-dijkstra-7d02a9.netlify.com/

### CODE REVIEW

File: gatsby-site/src/components/BackgroundWave/BackgroundWave.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.wave-container {
+    position: absolute;
+}
+
+.purple-wave {
+    top: -550px;
+    z-index: -2 !important;
+    position: absolute;
```
Comment (alodahl) [inline gatsby-site/src/components/BackgroundWave/BackgroundWave.css]: purple and blue wave will inherit absolute from their parent, wave-container.  No need to type it more than once.

**Comment by lianthompson** [comment]
https://5b7f3eb302ed8325f5fc73a1--relaxed-dijkstra-7d02a9.netlify.com/

**Comment by lianthompson** [comment]
https://5b7f3eb302ed8325f5fc73a1--relaxed-dijkstra-7d02a9.netlify.com/

**Comment by alodahl** [comment]
Here's the deploy preview with height-responsive background waves:

https://deploy-preview-13--relaxed-dijkstra-7d02a9.netlify.com/

**Pull request merged**